lib/fluent/plugin/in_prometheus_tail_monitor.rb

```diff
@@ -72,9 +81,13 @@ def update_monitor_info
           # Access to internal variable of internal class...
           # Very fragile implementation
           pe = watcher.instance_variable_get(:@pe)
+          monitor_info = watcher.instance_variable_get(:@metrics)
```

Comment:
Please add a guard for the case `monitor_info == nil`.
Fluentd v1.14.0 or former don't have this feature.
In this case, the plugin will crash.
Users might upgrade this plugins without upgrading Fluentd.

Reply:
Thanks for the review, I've added a guard.